### ISSUE TITLE
"not equal sign" didn't render properly

### ISSUE THREAD

**Issue by azophy** [obedm503/katex-latex#16]
When I tried to write `\ne` in latex format or `!=` in asciimath format, the strike in the equal sign was misaligned. This happen even on the demo page on https://obedm503.github.io/showdown-katex/#sandbox 

![2020-03-08_55](https://user-images.githubusercontent.com/2698551/76158245-5df28000-6146-11ea-9d79-f2f072c30be2.png)

**Comment by obedm503** [comment]
I've confirmed this is an issue. I tried quickly figuring out what may be causing it, but have not been able to. It seems to be related to KaTeX/KaTeX#1842. Any help is appreciated.

Until this issue is fixed, you can use [`\cancel{=}`](https://obedm503.github.io/showdown-katex?source=```asciimath%0Aa+\cancel{%3D}+b%0A```%0A```latex%0Aa+\cancel{%3D}+b%0A```) instead (even in asciimath blocks).

What browser and OS are you using?

**Comment by lriuui0x0** [comment]
It's interesting that if we use `renderMathInElement` from katex directly, this error doesn't seem to occur.

**Comment by obedm503** [comment]
@lriuui0x0 can you provide some more details? you mentioned using `renderMathInElement` directly, but this package [already uses `renderMathInElement`](https://github.com/obedm503/showdown-katex/blob/a37754d7654734cf457fa07a8b69b7fc2c55628b/src/showdown-katex.js#L95)

I've been looking into this issue without much success. how did you get `renderMathInElement` to work?

**Comment by lriuui0x0** [comment]
@obedm503 Yep I had code like below to render maths. `code.math` is the class that's assigned when you do ```math with showdown.

```
Array.from(text_root.querySelectorAll('code.math')).forEach(code => {
    let p = document.createElement('p');
    katex_html = katex.renderToString(code.innerText, {
        displayMode: true,
        throwOnError: false,
    });
    p.innerHTML = katex_html;
    code.replaceWith(p);
});
renderMathInElement(text_root, {
    throwOnError: false,
    delimiters: [
        {left: "$", right: "$", display: false},
    ]
});
```

**Comment by obedm503** [comment]
what version of katex are you using?

**Comment by lriuui0x0** [comment]
@obedm503 Latest. I'm using `<script src="https://cdn.jsdelivr.net/npm/katex/dist/katex.min.js"></script>`
